### ISSUE TITLE
Hard to follow through the authorization architecture.

### ISSUE THREAD

**Issue by KarthikRangaraju** [spinnaker/spinnaker.github.io#2015]
Page: [/reference/architecture/authz_authn/authorization/](https://www.spinnaker.io/reference/architecture/authz_authn/authorization/)

Feedback:
No diagrams, no basic introduction of components/terminologies. Goes straight into random details. There is no head, there is no tail. Feeling lost with this page.

**Comment by dorbin** [comment]
Hi, @KarthikRangaraju , thanks for the feedback. 
The page you're looking at is a reference document, and it could probably stand some improvement. But there's also some setup documentation for authorization. Have a look at [this](https://spinnaker.io/setup/security/authorization/) and see if it's more helpful. 

Cheers,
Dave

**Comment by dorbin** [comment]
@KarthikRangaraju , did the documentation I pointed to work better? I'm closing this issue, but feel free to reopen it if the linked doc doesn't work.
